src/expressionEditor.js

```diff
@@ -72,8 +72,8 @@ define([
             richTextOptions = {isExpression: true},
             form = options.mug.form;
         options = _.defaults(options, {
-            leftPlaceholder: "Hint: drag a question here.",
-            rightPlaceholder: "Hint: drag a question here.",
+            leftPlaceholder: "Drag question here",
+            rightPlaceholder: "Drag question here",
```

Comment:
https://github.com/dimagi/Vellum/commit/94462c05777ea458c98a68abb6073be6d7f8d885

(I'm going for as few characters as possible, as the inputs are now a little smaller)